.circleci/config.yml

```diff
@@ -94,11 +93,24 @@ jobs:
       - restore_cache:
           keys:
             - go-mod-test-v1-{{ checksum "go.sum" }}
-      - run: go test -v
+      - run:
+          name: Run go tests
+          command: |
+            mkdir -p /tmp/test-results
+            gotestsum -f standard-verbose --junitfile ${TEST_RESULTS}/results.xml .
```

Comment:
Should this format just be consistent with the other results? I don't care too much about which format we use 😄

Reply:
Ya, that's a good idea. I will make that change in a follow up with a few other changes.